src/components/CreationProgress/index.tsx

```diff
@@ -43,7 +50,9 @@ const sizes = {
     stepTextMargin: 15,
     text: 10,
   },
-}
+} as const
```

Comment:
Thanks, I've learn something
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions
but I think it's not usefull here (we don't need the sizes values in types) ?

```suggestion
} 
```